python-service/src/node_registry.py

```
@@ -0,0 +1,49 @@
+import json
+import urllib.request
+import urllib.error
+from typing import Any, Dict, List, Optional
```

Flake8 (7.3.0)

[error] 'typing.Optional' imported but unused

(F401)